app/assets/javascripts/admin/addon/components/webhook-events.js

```diff
@@ -11,10 +11,17 @@ import I18n from "discourse-i18n";
 export default class WebhookEvents extends Component {
   @service messageBus;
   @service store;
+  @service dialog;
 
   @tracked pingEnabled = true;
   @tracked events = [];
   @tracked incomingEventIds = [];
+  @tracked loading = false;
+  @tracked showProgress = false;
+  @tracked processedTopicCount = 0;
+  @tracked count = 0;
```

Comment:
You don't need the `count` defined up here. You can simply set `const count = this.eventIds.length;` on line 48 below. No need to set it on the component.

Reply:
Oh I think this variable was used for multiple things.. Can you adjust the naming so that it is more clear? Like maybe it is `failedEventCount`